### PR TITLE
Remove runtime-checks

### PR DESCRIPTION
**Pull request by jonathanKingston**
Task: https://app.asana.com/0/1201614831475344/1208341763117531/f

Removes the runtime-checks feature and all it's related features.

### CODE REVIEW

**Review by shakyShane — APPROVED** [review]
I havn't checked if there's anything missing, but the tests are passing and it lgtm.

**Comment by shakyShane** [comment]
@jonathanKingston just a heads-up that I already updated this branch with main, so that I could build off it.

**Comment by shakyShane** [comment]
* **#1084** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1084?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1089** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1089?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1066** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1066?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jonathanKingston and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: integration-test/playwright/remote-pages.spec.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ function getHARPath (harFile) {
     return path.join(testRoot, 'data', 'har', harFile)
 }
 
-const config = './integration-test/test-pages/runtime-checks/config/replace-element.json'
 const css = readFileSync('./build/integration/contentScope.js', 'utf8')
-const parsedConfig = JSON.parse(readFileSync(config, 'utf8'))
+// TODO flag on all features
+const parsedConfig = {}
```
Comment (jonathanKingston) [inline integration-test/playwright/remote-pages.spec.js:13]: @shakyShane probs relevant for your work on top of this btw.

**Review by shakyShane — APPROVED** [review]
thank you!

**Pull request merged**